Dockerfile

```diff
@@ -26,7 +28,9 @@ RUN mkdir -p /tmp/gcloud && \
 
 	cd && rm -rf /tmp/gcloud
 
-COPY build/drone-gcloud-helm /opt/google-cloud-sdk/bin/
+COPY plugin.go main.go ./
+
+RUN mkdir /go && go get && go install && go build && mv /go/bin/_ /opt/google-cloud-sdk/bin/drone-gcloud-helm
```

Comment:
Compiled die `main.go` naar `/go/bin/_` ofzo? Ik volg em hier even niet.

Reply:
ja dat is lame idd.
hij compiled idd naar /go/bin/_ :see_no_evil: 

Hij pakt de map warin het zit.
Dus ~/devel/drone-gcloud-helm zou naar $GOPATH/bin/<foldername> gaan, dwz `$GOPATH/bin/drone-gcloud-helm`

Maar hier wordt in de container root gewerkt :see_no_evil: omdat ik hier ook zat te klungelen.
En dan is het blijkbaar `$GOPATH/bin/_`.

Ik weet niet in hoever wij drone op publieke repo's willen (dit is een forked repo)
Er zijn mogelijkheden builds te inhibiten maar ik durfde het niet, moeten we misschien een keer evalueren.
Men kan de .drone.yml signieren zodat het niet gebruikt kan wordem om say onze docker upload secret te lekken.

Dus omdat ik het werkend had en de binary gevonden heb was ik al lang blij